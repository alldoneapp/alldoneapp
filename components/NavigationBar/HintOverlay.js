import React, { Component } from 'react'
import { StyleSheet, View } from 'react-native'
import { colors } from '../styles/global'

export default class HintOverlay extends Component {
    constructor(props) {
        super(props)
    }

    render() {
        return (
            <View style={localStyles.container}>
                <View style={localStyles.redDisk}></View>
            </View>
        )
    }
}

const localStyles = StyleSheet.create({
    container: {
        height: 12,
        width: 12,
        borderRadius: 100,
        backgroundColor: 'white',
        alignItems: 'center',
        justifyContent: 'center',
    },
    redDisk: {
        height: 8,
        width: 8,
        backgroundColor: colors.Red200,
    },
})
