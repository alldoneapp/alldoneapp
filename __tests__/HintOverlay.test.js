import React from 'react'
import HintOverlay from '../components/NavigationBar/HintOverlay'

import renderer from 'react-test-renderer'

describe('HintOverlay component', () => {
    describe('HintOverlay snapshot test', () => {
        it('should render correctly', () => {
            const tree = renderer.create(<HintOverlay />).toJSON()
            expect(tree).toMatchSnapshot()
        })
    })
})
