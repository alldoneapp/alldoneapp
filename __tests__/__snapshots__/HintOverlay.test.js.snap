// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HintOverlay component HintOverlay snapshot test should render correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "borderRadius": 100,
      "height": 12,
      "justifyContent": "center",
      "width": 12,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#E00000",
        "height": 8,
        "width": 8,
      }
    }
  />
</View>
`;
